manager/src/main/resources/db/migration/0.1/V0.1.21__create_processing_error.sql

```diff
@@ -0,0 +1,12 @@
+create sequence PROCESSING_ERROR_ID_SEQUENCE;
+
+create table PROCESSING_ERROR
+(
+    id                 bigint       NOT NULL PRIMARY KEY,
+    bridge_id          varchar(255) NOT NULL,
```

Comment:
This functionality is currently in the `manager` for the sake of time-to-release, but it's completely indipendent from the existing functionalities of the `manager` and in the future it may be moved outside of it (this is also the reason why I moved all the classes in a dedicated package).

For this reason I don't think we need to add this relation. The feature works perfectly without it.

Reply:
+1 - I guess some orphan records will remain in the table when a brigde gets deleted or removed